config/components/config/holiday.py

```diff
@@ -0,0 +1,12 @@
+from django_components import Component, register
+
+
+@register("config.holiday_table")
+class HolidayTable(Component):
+
+    # This component takes one parameter, a date string to show in the template
+    def get_context_data(self, holidays=""):
```

Comment:
I'd use 'None' as default value then, perhaps with a type-hint as well.

Reply:
fixed the type hint